Makefile

```diff
@@ -79,8 +82,19 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+operator-sdk:
+	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
+	chmod 0755 operator-sdk
```

Comment:
Here you could assign the `operator-sdk` to a variable, and use it in the other make targets.
```
OPERATOR_SDK=./operator-sdk 
```

Reply:
I think we don't have to change whole makefile only because travis.